app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt

```diff
@@ -225,36 +224,6 @@ class StoryFragmentTest {
     }
   }
 
-  @Test // TODO(#4212): Error -> Only the original thread that created a view hierarchy can touch
-  // its view
-  fun testStoryFragment_toolbarTitle_marqueeInRtl_isDisplayedCorrectly() {
```

Comment:
We should still be testing that the marquee works as expected. These tests should be updated rather than removed. Ditto for all other test suites & affected screens in this PR.

Reply:
Will revert the tests for this PR and find out the exact reason why they were failing.